tox.ini

```diff
@@ -0,0 +1,37 @@
+# If "pip" is not installed, install it running following command:
+#   $ yum install python-pip
+#
+# If "tox" is not installed, install it running following command:
+#   $ pip install -e git://github.com/tox-dev/tox.git@3.23.1#egg=tox
+#
+# After it you can use "tox" command. Example:
+#   $ tox -e py39 -- pwd
+
+[tox]
+# With version 1.6.0 'skipsdist' config option was added. It allows to skip
+# installation of current project to 'sdist' (no req to define setup.py file).
+minversion = 1.6.0
+skipsdist = True
+sitepackages = False
+
+[testenv]
+envdir = {toxworkdir}/{envname}
+passenv = PYTEST_* SCT_*
+whitelist_externals = *
+commands =
+    python -m pip install --upgrade pip>=9.0.0 setuptools wheel
+    pip install -r requirements.in
```

Comment:
Why not requirements.txt?

Reply:
If we use "requirements.txt" then we need to update it's hashes either by running pip-compile manually or using our "docker build and push" script which does it for the hydra image.
Using "requirements.in" file we can just add new dependency we want to try out developing some feature, not needing to do anything else except adding one line to that file.
Moreover, usage of the ".txt" variant takes about 5 minutes to be completed on my machine where just 1 minutes is needed for the ".in" variant.